### DISCUSSION TITLE
About CMI's MySQL Configuration Synchronization

### DISCUSSION THREAD

**Discussion by 8MiYile** [Zrips/CMI#7512] [Q&A]
When will CMI be able to do synchronous configuration on multiple servers via MySQL?

**Comment by MrCrazys** [comment]
> When will CMI be able to do synchronous configuration on multiple servers via MySQL?

From what I know, there is an interest by Zrips to add this, however being such a big plugin, it's bound to take a while to move all the features over. I of course have no ETA on this.
